docs/Installation/Installation_Guide.rst

```diff
@@ -1,579 +1,581 @@
+the **General Cluster Requirements** and skip to **Installing Zenko**, below.
+
+Otherwise, you can set up a cluster quickly using MetalK8s_, Scality's
+open-source Kubernetes cluster project, as described in "Setting Up a Metal K8s
+Cluster," below.
+
+The following section describes general cluster requirements, which are tested
+on Metal K8s. Because MetalK8s is designed to operate without support from
+public cloud resources, the following sizing requirements are assumed good for
+other cloud Kubernetes deployments, where such resources are preinstalled and
+available on demand.
+
+####################
+Setting Up a Cluster
+####################
+
+****************************
+General Cluster Requirements
+****************************
+
+Setting up a testing cluster requires at least three machines (these can be
+VMs) running CentOS_ 7.4 (The recommended mimimum for Zenko production service
+is five server nodes with three masters/etcds, but for testing and
+familiarization, three masters and three nodes is fine). You must have SSH
+access to these machines and they must have SSH access to each other. (You
+can copy SSH credentials from one machine to the next and log in once to
+ensure each machine has been added to the others' recognized hosts lists).
+Each machine acting as a Kubernetes_ node must also have at least one disk
+available to provision storage volumes.
+
+
+.. _MetalK8s: https://github.com/scality/metal-k8s/
+.. _CentOS: https://www.centos.org
+.. _Kubernetes: https://kubernetes.io
+
+
+Sizing
+======
+
+Sizing for Metal K8s
+--------------------
+
+-  Cores
+
+   -  2 for etcd
+   -  4 for master
+   -  4 for node
+
+-  RAM
+
+   -  4 GB for etcd
+   -  8 GB for master
+   -  8 GB for node
+
+-  Storage (for the system)
+
+   -  20 GB for the root filesystem
+   -  16 GB for etcd
+   -  128 GB for master
+   -  64 GB for node
+
+Sizing for Zenko
+----------------
+
+-  Cores
+
+   -  1 per node
+
+-  RAM
+
+   -  16 GB per node
+
+-  Storage
+
+   -  200 GB per node
+
+   This requirement is not for the system device. You may have to attach a
+   separate storage volume to your cloud server instance.
+
+All servers must be in CentOS 7.4, and accessible with ssh.
+
+Proxies
+=======
+
+If you are behind a proxy, add the following lines to your local machine’s
+/etc/environment file:
+
+::
+
+    http_proxy=http://user;pass@<my-ip>:<my-port>
+    https_proxy=http://user;pass@<my-ip>:<my-port>
+    no_proxy=localhost,127.0.0.1,10.*
+
+******************************
+Setting Up a Metal K8s Cluster
+******************************
+
+MetalK8s provides a stable, easy-to-deploy base for you to test Zenko in a
+live Kubernetes cluster environment.
+
+
+Clone or Copy the MetalK8s Git Repo
+===================================
+
+Either log in to your primary machine (any of the nodes can be the primary)
+and clone the metal-k8s repo directly from GitHub:
+
+.. code-block:: shell
+
+   $ git clone https://github.com/scality/metal-k8s

```

Comment:
Using this, users would be installing some random in-development version of MetalK8s, rather than the one we want to have them deploy, so this is not the right approach to 'download' MetalK8s.

Reply:
I'd recommend downloading the release files rather than pulling things from Git, for 'stable' installations, so https://github.com/scality/metalk8s/releases

Reply:
Done.